mass_operation_abstract/views/view_mass_operation_wizard_mixin.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2018-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+
+    <record id="view_mass_operation_wizard_mixin_form" model="ir.ui.view">
+        <field name="model">mass.operation.wizard.mixin</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
+                <p class="oe_grey" attrs="{'invisible':
+                    [('message', '=', False)]}">
+                    <field name="message"/>
+
+                </p>
+                <p class="alert alert-info" role="alert" attrs="{'invisible':
+                    [('operation_description_info', '=', False)]}">
+                    <field name="operation_description_info"/>
+                </p>
+
+                <p class="alert alert-warning" role="alert" attrs="{'invisible':
+                    [('operation_description_warning', '=', False)]}">
+                    <field name="operation_description_warning"/>
+                </p>
+
+                <p class="alert alert-danger" role="alert" attrs="{'invisible': [('operation_description_danger', '=', False)]}">
+                    <field name="operation_description_danger"/>
+                </p>
+
+                <group name="custom_info">
+                </group>
+                <footer>
+                    <button string="Apply" name="button_apply" type="object" class="oe_highlight"/>
+                    <button string="Cancel" class="oe_link" special="cancel"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+</odoo>
+
+
+<!--
+
+
+
+
+ -->
```

Comment:
:thinking: :question:

Reply:
@simahawk Can you remove this comment so we can move forward?

Reply:
WTH does it come from?? :thinking: :rofl: 
Done :wink: